config/vhosts/vhosts.inc

```diff
@@ -65,49 +82,45 @@ function vhosts_sync_package() {
 	global $config;
 
 	if ($config['installedpackages']['vhosts']['config'] != "") {
-		conf_mount_rw();
+		// # NOTE not needed anymore
+//		conf_mount_rw();
 
-		//sort the vhosts array
 		$vhostarray = $config['installedpackages']['vhosts']['config'];
-		if (count(vhostarray) > 1) {
-			usort($vhostarray, 'sort_ipaddress');
-			usort($vhostarray, 'sort_host');
-			usort($vhostarray, 'sort_port');
-		}
+		// # TODO remove loop to acquire enabled vhosts
+		// # NOTE same work can be done within loop to build conf file
 		$vhostarray_http = '';
-		$vhostarray_https = '';
+		// # HACK setting the Default port
+		$server_port = '8000';
```

Comment:
8000+ Is used by Captive Portal anyhow (2 ports per zone). Don't use anything near that. Use something much higher or much lower as the base value.